component-runtime-impl/src/main/java/org/talend/sdk/component/runtime/record/RecordImpl.java

```diff
@@ -587,11 +591,15 @@ private <T> Builder append(final Schema.Entry entry, final T value) {
             } else {
                 realEntry = entry;
             }
-            if (value != null) {
-                values.put(realEntry.getName(), value);
-            } else if (!realEntry.isNullable()) {
-                throw new IllegalArgumentException(realEntry.getName() + " is not nullable but got a null value");
-            }
+//            if (Boolean.parseBoolean(System.getProperty("talend.sdk.skip.nullable.check", "false"))) {
+//                values.put(realEntry.getName(), value);
+//            } else {
+                if (value != null) {
+                    values.put(realEntry.getName(), value);
+                } else if (!realEntry.isNullable()) {
+                    throw new IllegalArgumentException(realEntry.getName() + " is not nullable but got a null value");
+                }
```

Comment:
same code as before, no check is done.

Reply:
For this issue, only need to add the check in build(), but not in validateTypeAgainstProvidedSchema, discussed with wwang and confirmed from him. 
It is used only for build, for validation it is still needed.